velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -887,17 +887,17 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(getUnixTime("2020-06-30 23:59:59"), "yyyy-MM-dd HH:mm:ss"),
       "2020-06-30 23:59:59");
 
+  // 8 hours ahead UTC.
+  setQueryTimeZone("Asia/Shanghai");
 // In debug mode, Timestamp constructor will throw exception if range check
 // fails.
 #ifdef NDEBUG
   // Integer overflow in the internal conversion from seconds to milliseconds.
   EXPECT_EQ(
       fromUnixTime(std::numeric_limits<int64_t>::max(), "yyyy-MM-dd HH:mm:ss"),
-      "1969-12-31 23:59:59");
+      "1970-01-01 07:59:59");
```

Comment:
It would be better to also keep the original test where default timezone is used.

Reply:
Thanks. It's done in https://github.com/facebookincubator/velox/pull/9836/commits/225d1f272d2d9b1f0d09d36cf64bb9f9d5df4674